### PR TITLE
Adds 'Need help' button

### PR DESCRIPTION
**Pull request by jordanwade**
This adds the find group help button. I also tweaked/iterated on a few things in the header area of the results page.
image

### CODE REVIEW

File: components/GroupSearchFilters/GroupSearchFilters.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ import PropTypes from 'prop-types';
 
 import { useModalDispatch, showModal } from 'providers/ModalProvider';
 import { useGroupFilters, resetValues } from 'providers/GroupFiltersProvider';
-import { Box, Button, Divider } from 'ui-kit';
+import { Box, Button, Divider, Icon } from 'ui-kit';
+import { rem } from 'ui-kit/_utils';
```
Comment (drewbarontini) [inline components/GroupSearchFilters/GroupSearchFilters.js:6]: You can do `import { Box, Button, Divider, Icon, utils } from 'ui-kit';` and then `utils.rem()`.

File: ui-kit/_config/icons.js [hunk 2]
```diff
@@ -13,6 +13,8 @@ const icons = {
...
+  filter:
+    'M1.44926 6.72463C0.599034 6.72463 0 6.18359 0 5.37199C0 4.54107 0.599034 4 1.44926 4H22.5315C23.3818 4 24 4.54107 24 5.37199C24 6.18359 23.3818 6.72463 22.5315 6.72463H1.44926ZM4.3285 13.1208C3.45893 13.1208 2.8599 12.5797 2.8599 11.7682C2.8599 10.9372 3.45893 10.3962 4.3285 10.3962H19.6715C20.5219 10.3962 21.1208 10.9372 21.1208 11.7682C21.1208 12.5797 20.5219 13.1208 19.6715 13.1208H4.3285ZM7.30436 19.5362C6.45413 19.5362 5.8551 18.9951 5.8551 18.1836C5.8551 17.3527 6.45413 16.8116 7.30436 16.8116H16.6764C17.5266 16.8116 18.145 17.3527 18.145 18.1836C18.145 18.9951 17.5266 19.5362 16.6764 19.5362H7.30436Z',
```
Comment (drewbarontini) [inline ui-kit/_config/icons.js:17]: Nitpick: Alphabetize 😬

**Review by drewbarontini — COMMENTED** [review]
Diggin' it! The only thing throwing me off in the header is the "Clear" link feels like it's just floating out there. But we can easily continue to iterate on that, and maybe @rkdavidson has further thoughts…

File: components/GroupSearchFilters/GroupSearchFilters.js (hunk 1 above)
Comment (rkdavidson) [reply components/GroupSearchFilters/GroupSearchFilters.js:6]: That's good to know, I did this `_utils` import in another spot. Didn't bother to look to see if there was a more convenient import...

File: components/GroupSearchFilters/GroupSearchFilters.js [hunk 3]
```diff
@@ -20,55 +21,64 @@ function GroupSearchFilters(props = {}) {
...
-          )}
-        </Box>
-        <Box as="p" fontWeight="bold">{`${props.resultsCount} ${
-          props.resultsCount === 1 ? 'group' : 'groups'
-        }`}</Box>
-      </Box>
-
+      <Box as="p" fontWeight="bold">{`${props.resultsCount} ${
+        props.resultsCount === 1 ? 'group' : 'groups'
+      }`}</Box>
```
Comment (rkdavidson) [inline components/GroupSearchFilters/GroupSearchFilters.js:80]: TBH, I'm not sure how much value there is in the total count at the moment.
A.) It's inaccurate, and B.) it feels awkward in the UX now.

@jordanwade, (cc: @drewbarontini ) WDYT if we just remove it for now?
```suggestion
```

**Review by rkdavidson — DISMISSED** [review]
LGTM, thanks for knocking this out 🙌

**Comment by rkdavidson** [comment]
@jordanwade and @drewbarontini , WDYT about this re: Drew's feedback on the floating/dangling Clear button?

In the all filters modal, we could put a **Clear All** secondary button in bottom left?
The language and/or sizing of buttons could be refined, but this feels like a step in the right direction to me.

image

**Comment by drewbarontini** [comment]
@rkdavidson I'm good with that 👍

**Review by rkdavidson — APPROVED** [review]
LGTM 👍 Thanks for updating those buttons in the All Filters modal.

**Pull request merged**